src/kokkos/test/kokkos/AtomicPairCounter_t.cc

```diff
@@ -0,0 +1,75 @@
+#include "KokkosCore/kokkosConfigCommon.h"
```

Comment:
After #47 this needs to be changed to
```suggestion
#include "KokkosCore/InitializeScopeGuard.h"
```